### PR TITLE
Department integration tests

### PR DESCRIPTION
**Pull request by bgregg**
^^RT

### CODE REVIEW

File: spec/features/department_creation_spec.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+describe 'creating departments' do
+  before :each do
+    when_current_user_is :staff, integration: true
+    visit new_department_path
+  end
+
+  let!(:department_fields) { attributes_for(:department).except :staff }
```
Comment (dfaulken) [inline spec/features/department_creation_spec.rb]: Departments don't have a `staff` attribute...

File: spec/features/department_deletion_spec.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe 'deleting users' do
+  before :each do
+    when_current_user_is :staff, integration: true
+    @dept = create :department
```
Comment (dfaulken) [inline spec/features/department_deletion_spec.rb]: I'd prefer, before your `before :each` block:

```ruby
let(:department) { create :department }
```

... and then you can refer to it everywhere as a local variable.

In general, prefer `let` blocks to instance variables (source: https://github.com/nevir/rubocop-rspec/blob/master/lib/rubocop/cop/rspec/instance_variable.rb#L6-L20)

**Comment by dfaulken** [comment]
Looks great besides my comments. You have the go-ahead to ignore the codeclimate failure.

**Comment by dfaulken** [comment]
Pursuant to https://github.com/umts/jobapps/pull/180, please merge and use the updated `fill_in_fields_for` method where applicable.

**Comment by bgregg** [comment]
The ```fill_in_fields_for``` method was already in use in the department spec. I believe it was just the position spec that was giving us trouble.

**Comment by dfaulken** [comment]
Yes, my mistake. This is good to go.

**Pull request merged**